.buildkite/install-lmcache.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+CONDA_ENV_NAME="buildkite"
+PYTHON_VERSION=3.10
+
+cuda_version=12.1
+export CUDA_HOME=/usr/local/cuda-${cuda_version}
+export LD_LIBRARY_PATH=$CUDA_HOME/lib64:$LD_LIBRARY_PATH
+export PATH=$CUDA_HOME/bin:$PATH
+
+eval "$(conda shell.bash hook)"
+conda activate ${CONDA_ENV_NAME}
+
+set -xe 
+
+pip install -e third_party/torchac_cuda
+pip install -e .
```

Comment:
Consider using ccache to cache some compile work --- compile is slow.

Reply:
Added a new ticket #43 for this. Also, will do this in v0.1.x